### ISSUE TITLE
matplotlib >= 3.0 compatibility

### ISSUE THREAD

**Issue by dbkeitel** [PyFstat/PyFstat#31]
In #29 @Rodrigo-Tenorio noticed problems:
> Moreover, corner plot requires matplotlib < 3.0 due to deprecated options being used.

However, for myself with python 3.7.3 and matplotlib 3.1.1 there seems to be no problem with corner plots e.g. in `examples/followup_examples/semi_coherent_directed_follow_up.py`

Rodrigo, can you track down what exact options were causing you problems, and which version you were using?

**Comment by dbkeitel** [comment]
@Rodrigo-Tenorio Was this 300f839e6830d673f4afa31f5cf87ea5662240be (hist "normed" argument) the only deprecation issue you saw?

The message it gave me was
```
The 'normed' kwarg was deprecated in Matplotlib 2.1 and will be removed in 3.1. Use 'density' instead.
```
so it's actually weird that it was still working for me in 3.1.3. But anyway, I've updated the one line calling it and bumped the minimum to 2.1 to be on the safe side. (In practice we probably require higher versions for some other features, but the requirements aren't carefully vetted for minimum versions in general anyway.)

If you have any other incompatibility with a version >3.0, please let me know, and otherwise you can close the issue.

**Comment by Rodrigo-Tenorio** [comment]
Well, I just run a test follow up using matplotlib 3.1.3 and nothing happened. My bad I did not specify the exact error I got. 

I recall it was related to one of the functions provided by corner, but probably the actual kwarg was comming from PyFstat and is already fixed by 300f839.

If this is the case, we can go back to matplotlib >= 3.0
